drivers/adc/adc_npcx.c

```diff
@@ -52,6 +49,8 @@ struct adc_npcx_config {
 	struct npcx_clk_cfg clk_cfg;
 	/* pinmux configuration */
 	const struct npcx_alt *alts_list;
+	/* amount of thresholds supported */
+	uint8_t threshold_count;
```

Comment:
As the threshold_count is fixed, how about add the **const** qualifer?

Reply:
Done.